os/pm/Kconfig

```diff
@@ -42,6 +39,63 @@ config PM_METRICS_DURATION
 
 endif
 
+comment "PM Options"
+
+config PM_NSTATE
+	int "Number of SoC sleep states"
+	default ""
+	---help---
+		Number of sleep states provided by the SoC Power module
+
+if PM_NSTATE != 0
+config PM_STATE_SWITCH_THRESH
```

Comment:
Please consider changing this to PM_STATE_SWITCH_DURATION or PM_STATE_ENTRY_DURATION.